tests/integrationv2/test_signature_algorithms.py

```diff
@@ -0,0 +1,176 @@
+import copy
+import pytest
+
+from configuration import available_ports, ALL_TEST_CIPHERS, ALL_TEST_CURVES, ALL_TEST_CERTS, PROVIDERS, PROTOCOLS
+from common import ProviderOptions, Protocols, Ciphers, Certificates, Signatures, data_bytes
+from fixtures import managed_process
+from providers import Provider, S2N, OpenSSL
+from utils import invalid_test_parameters, get_parameter_name, get_expected_s2n_version
+
+
+certs = [
+    Certificates.RSA_2048_SHA256,
+    Certificates.RSA_2048_SHA384,
+    Certificates.RSA_PSS_2048_SHA256,
+    Certificates.ECDSA_256,
+    Certificates.ECDSA_384,
+]
+
+all_sigs = [
+    Signatures.RSA_SHA1,
+    Signatures.RSA_SHA224,
+    Signatures.RSA_SHA256,
+    Signatures.RSA_SHA384,
+    Signatures.RSA_SHA512,
+    Signatures.ECDSA_SECP256r1_SHA256,
+    Signatures.RSA_PSS_SHA256,
+]
+
+unsupported_ciphers = [
```

Comment:
Why are these ciphers unsupported, and are they only unsupported for this test?

Reply:
Maybe unsupported is not really correct. I've updated the comment here. For some reason s_client doesn't show use the peer signing information when these ciphers are used. These can be re-enabled when #1735 is completed.